recipes-graphics/libegt/libegt/0001-link-libudev-for-udev_unref-and-udev_new.patch

```diff
@@ -0,0 +1,37 @@
+From 8ffc0ab37d9bf7cc46fa9fa27cc1eb7f88f554b1 Mon Sep 17 00:00:00 2001
```

Comment:
Could you please send this patch to https://github.com/linux4sam/egt ? Thanks!

Reply:
done https://github.com/linux4sam/egt/pull/1
but lets apply this patch in meta layer until its accepted upstream

Reply:
Can you apply patch 1 and patch 3 then from this series ? and leave patch 2 out ?

Reply:
Sure thing. Thanks!